components/home/three-campuses.js

```diff
@@ -1,61 +1,59 @@
-import { Card } from "@/components/card";
+import { Card, CardImage, CardContent, CardTitle } from "@uoguelph/react-components/card";
 import guelph from "@/img/guelph.png";
 import ridgetown from "@/img/ridgetown.png";
 import guelphHumber from "@/img/guelph-humber.png";
+import { UnstyledLink } from "@/components/unstyled-link";
+import Image from "next/image";
+import { tv } from "tailwind-variants";
 
 export const ThreeCampuses = () => {
-  const campuses = [
-    {
-      title: "Guelph",
-      url: "https://www.uoguelph.ca/admission/undergraduate/tours/",
-      image: {
-        src: guelph.src,
-        width: guelph.width,
-        height: guelph.height,
-        blurred: guelph.blurDataURL,
-        alt: "An aerial view of Johnston Hall and the University of Guelph campus",
-      },
+  const classes = tv({
+    slots: {
+      container: "grid grid-cols-1 gap-4 sm:grid-cols-2 md:grid-cols-3",
+      card: "h-full",
+      image: "aspect-3/2 w-full",
+      content: "flex-1",
+      title: "text-[2.25rem]! font-bold text-center",
     },
-    {
-      title: "Ridgetown",
-      url: "https://www.ridgetownc.com/",
-      image: {
-        src: ridgetown.src,
-        width: ridgetown.width,
-        height: ridgetown.height,
-        blurred: ridgetown.blurDataURL,
-        alt: "An aerial view of the Ridgetown Campus",
-      },
-    },
-    {
-      title: "Guelph-Humber",
-      url: "https://www.guelphhumber.ca/",
-      image: {
-        src: guelphHumber.src,
-        width: guelphHumber.width,
-        height: guelphHumber.height,
-        blurred: guelphHumber.blurDataURL,
-        alt: "The main building of the University of Guelph-Humber campus",
-      },
-    },
-  ];
+  });
+
+  const { container, card, image, content, title } = classes();
 
   return (
-    <div className="grid grid-cols-1 gap-4 sm:grid-cols-2 md:grid-cols-3">
-      {campuses.map((campus, index) => (
-        <Card
-          key={campus.title}
-          className="h-full"
-          title={<span className="my-auto w-full text-center text-[2.2rem] font-bold">{`${campus.title} Campus`}</span>}
-          href={campus.url}
-          centered
-          image={{
-            ...campus.image,
-            className: "aspect-[3/2] w-full",
-            sizes: "(max-width: 640px) 100vw, (max-width: 1024px) 50vw, 33vw",
-          }}
+    <div className={container()}>
+      <Card as={UnstyledLink} href="https://www.uoguelph.ca/admission/undergraduate/tours/" className={card()} centered>
+        <CardImage
+          src={guelph}
+          className={image()}
+          as={Image}
+          alt="An aerial view of Johnston Hall and the University of Guelph campus"
```

Comment:
Same issue as spotlight cards - alt text should be null in a combined image + text link
<img width="1587" alt="image" src="https://github.com/user-attachments/assets/27ac6c33-9247-4dd2-842a-8de8c1cc3f96" />